.github/FUNDING.yml

```
@@ -0,0 +1,2 @@
+github: [tryallthetings]
+ko_fi: allthethings
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)